packages/browser-extension/src/scripts/dialog.js

```diff
@@ -205,9 +218,14 @@ async function submitButtonClickHandler(event) {
   const reason = Number.isNaN(reasonIndex) ? 'Unknown' : reasons[reasonIndex];
   const submitView = dialog?.getElementsByTagName('report-dialog-submit-view')[0];
   const userAgent = window.navigator.userAgent;
-
-  const issueUrl = await dispatch({ userAgent, reason, type: 'REPORT' });
-
+  const textareaValue = dialog?.querySelector('#explanation').value;
+  console.log('Textarea value:', textareaValue);
+  const issueUrl = await dispatch({
+    userAgent,
+    reason,
+    type: 'REPORT',
+    explanation: textareaValue,
```

Comment:
Please, switch lines between `explanation` and `userAgent`.

Reply:
done